selfswab/tasks.py

```diff
@@ -59,8 +65,6 @@ def poll_meditech_api_for_results():
                         "INVALID",
                         "Invalid",
                     ]
-                    if result == "":
```

Comment:
We don't want this record being updated because it is already `pending`, we also don't want to start the flow in Rapidpro for these.